pallets/committee/src/lib.rs

```diff
@@ -107,16 +108,14 @@ pub trait WeightInfo {
 pub type ProposalIndex = u32;
 
 /// The committee trait.
-pub trait Trait<I>: frame_system::Trait + IdentityModuleTrait {
-    /// The outer origin type.
-    type Origin: From<RawOrigin<<Self as frame_system::Trait>::AccountId, I>>;
-
-    /// The outer call dispatch type.
-    type Proposal: Parameter
-        + Dispatchable<Origin = <Self as Trait<I>>::Origin>
-        + GetDispatchInfo
-        + From<frame_system::Call<Self>>;
-
+pub trait Trait<I>:
+    frame_system::Trait<
+        Call: Parameter
+                  + Dispatchable<Origin = <Self as frame_system::Trait>::Origin>
+                  + GetDispatchInfo,
+        Origin: From<RawOrigin<<Self as frame_system::Trait>::AccountId, I>>,
+    > + IdentityModuleTrait
```

Comment:
I'm not convinced that this is more readable.

Reply:
This isn't for better readability, but removing associated types that can only be equated in one way, introducing unnecessary & duplicated configuration.